src/index.ts

```diff
@@ -146,6 +146,20 @@ async function main() {
       }`,
     )
   }
+  
+  const packageJsonPath = path.join(targetDirectoryPath, 'package.json')
+  const packageJson = fs.readFileSync(packageJsonPath, 'utf-8')
```

Comment:
Handle the error if a `pacakage.json` is not found. For example, the `deno` starter does not have `pacakge.json`.

Reply:
Sure man